proto/palomachain/paloma/skyway/msgs.proto

```diff
@@ -205,4 +218,26 @@ message MsgUpdateParams {
 
 // MsgUpdateParamsResponse defines the response structure for executing a
 // MsgUpdateParams message.
-message MsgUpdateParamsResponse {}
+message MsgUpdateParamsResponse {
+}
+
+// MsgLightNodeSaleClaim is sent by pigeon whenever it gets a sale notification
+// from compass
+message MsgLightNodeSaleClaim {
+  option (cosmos.msg.v1.signer) = "metadata";
+
+  palomachain.paloma.valset.MsgMetadata metadata = 1
+      [ (gogoproto.nullable) = false ];
+
+  uint64 event_nonce = 2;
+  uint64 eth_block_height = 3;
```

Comment:
What about

```suggestion
  uint64 remote_block_height = 3;
```

Reply:
Yes, I wrote this before your PR renaming the fields - https://github.com/palomachain/paloma/pull/1228

Should I merge to that branch instead? There may be some conflicts between the two.

Reply:
Oh, that's right, the interface is expecting this field - https://github.com/palomachain/paloma/blob/master/x/skyway/types/msgs.go#L117
So we either keep it for now, or merge your PR as well, or rename it after the PR is merged.

Reply:
Okay, I suggest we move this into the `dev16` branch as well then. It will block rolling this out before PF is ready, but it looks like there's some overlap anyway.

Reply:
I rebased to the `dev16` branch and fixed the conflicts. However, there are still some eth mentions in the codebase, most importantly the [EthereumClaim interface](https://github.com/palomachain/paloma/blob/master/x/skyway/types/msgs.go#L107). I could refactor it as well, but it would be a big change for both paloma and pigeon. I'm not sure if we should do it now under the current time constraints, but it should be addressed in its own ticket anyway.

Reply:
There's nothing we gain here apart from consistency. I value consistency pretty highly, but let's circle back to this one and focus on getting the whole thing out the door first.